### PR TITLE
fix(InventoryTable): RHICOMPL-3304 use shared tags filter instead of custom one

### PR DESCRIPTION
**Pull request by mkholjuraev**
Resolves: [RHICOMPL-3304](https://issues.redhat.com/browse/RHICOMPL-3304)

As we are working on making tags filter consistent, using the shared filter is required. This PR removes the implementation of a custom tag filter and adopts the shared filter from the InventoryTable component.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: src/SmartComponents/SystemsTable/SystemsTable.js [hunk 1]
```diff
@@ -86,25 +85,12 @@ export const SystemsTable = ({
...
-    },
-  });
-
   const constructedQuery = constructQuery(columns);
 
   const systemFetchArguments = {
     query: constructedQuery.query,
     variables: {
       ...constructedQuery.fragments,
-      tags: currentTags,
```
Comment (bastilian) [inline src/SmartComponents/SystemsTable/SystemsTable.js:107]: We will still need to pass the tags to the GraphQL API here. This probably needs to be adapted to use the tags passed in via the arguments for the `getEntities` function.

Comment (mkholjuraev) [reply src/SmartComponents/SystemsTable/SystemsTable.js:107]: I have verified that selected tags are being correctly [passed](https://github.com/RedHatInsights/compliance-frontend/blob/master/src/SmartComponents/SystemsTable/hooks.js#L164) into getEntities function by the InventoryComponent.

Comment (mkholjuraev) [reply src/SmartComponents/SystemsTable/SystemsTable.js:107]: NVM ^. I understand what you mean now.

**Comment by bastilian** [comment]
There seems to be an issue with the Tags dropdown, but I'm not sure if this is an issue that needs fixing here or in the Inventory: 

https://user-images.githubusercontent.com/7757/191920753-0467f8ce-3011-4caa-87dc-446e53dbf232.mp4

**Comment by mkholjuraev** [comment]
@bastilian it looks like the issue is on the Inventory side. I have observed the same issue on Advisor and patch also at least.

https://user-images.githubusercontent.com/59481011/191924238-ff5173f4-575f-48fc-872e-9a92f8f963cd.mp4

**Review by bastilian — APPROVED** [review]
Works for me! Thank you @mkholjuraev!

**Pull request merged**